homeassistant/components/motioneye/switch.py

```diff
@@ -18,19 +18,38 @@
 from homeassistant.components.switch import SwitchEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import EntityDescription
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from . import MotionEyeEntity, get_camera_from_cameras, listen_for_new_cameras
 from .const import CONF_CLIENT, CONF_COORDINATOR, DOMAIN, TYPE_MOTIONEYE_SWITCH_BASE
 
 MOTIONEYE_SWITCHES = [
-    (KEY_MOTION_DETECTION, "Motion Detection", True),
-    (KEY_TEXT_OVERLAY, "Text Overlay", False),
-    (KEY_VIDEO_STREAMING, "Video Streaming", False),
-    (KEY_STILL_IMAGES, "Still Images", True),
-    (KEY_MOVIES, "Movies", True),
-    (KEY_UPLOAD_ENABLED, "Upload Enabled", False),
+    EntityDescription(
```

Comment:
Please use the corresponding base entity description fo each platform.

https://github.com/home-assistant/core/blob/6eae5231f1f17760d9a4d3108ca5708beecf4a08/homeassistant/components/switch/__init__.py#L89